addons/zeus/stringtable.xml

```diff
@@ -6,102 +6,119 @@
             <Polish>Ustawienia Zeusa</Polish>
             <Spanish>Ajustes Zeus</Spanish>
             <Czech>Nastavení Zeuse</Czech>
+            <German>Zeus-Einstellungen</German>
         </Key>
         <Key ID="STR_ACE_Zeus_Settings_Description">
             <English>Provides control over various aspects of Zeus.</English>
             <Polish>Pozwala kontrolować różne aspekty Zeusa.</Polish>
             <Spanish>Proporciona control sobre diversos aspectos de Zeus.</Spanish>
             <Czech>Poskytuje kontrolu na různými aspekty Zeuse.</Czech>
+            <German>Bietet die Steuerung verschiedener Zeus-Optionen an.</German>
         </Key>
         <Key ID="STR_ACE_Zeus_ascension_DisplayName">
             <English>Ascension Messages</English>
             <Polish>Wiad. o nowym Zeusie</Polish>
             <Spanish>Mensajes de ascensión</Spanish>
             <Czech>Zpráva o novém Zeusovi</Czech>
+            <German>Aufstiegsnachrichten</German>
         </Key>
         <Key ID="STR_ACE_Zeus_ascension_Description">
             <English>Display global popup messages when a player is assigned as Zeus.</English>
             <Polish>Wyświetlaj globalną wiadomość kiedy gracz zostanie przydzielony jako Zeus</Polish>
             <Spanish>Mostrar mensajes emergentes globales cuando a un jugador se le asigna como Zeus.</Spanish>
             <Czech>Zobrazit globální zprávu když je hráč přiřazen jako Zeus.</Czech>
+            <German>Zeige globale Popup-Nachrichten wenn ein Spieler zu Zeus wird.</German>
         </Key>
         <Key ID="STR_ACE_Zeus_bird_DisplayName">
             <English>Zeus Eagle</English>
             <Polish>Orzeł Zeusa</Polish>
             <Spanish>Águila Zeus</Spanish>
             <Czech>Orel Zeuse</Czech>
+            <German>Zeus-Adler</German>
         </Key>
         <Key ID="STR_ACE_Zeus_bird_Description">
             <English>Spawn an eagle that follows the Zeus camera.</English>
             <Polish>Spawnuj orła, który podąrza za kamerą Zeusa.</Polish>
             <Spanish>Generar un águila que sigue la cámara Zeus.</Spanish>
             <Czech>Vytvoří orla, který následuje kameru Zeuse.</Czech>
+            <German>Spawne einen Adler, der der Zeus-Kamera folgt.</German>
```

Comment:
http://www.duden.de/suchen/dudenonline/spawnen

Keine Treffer zu `Spawnen` ... Bitte in korrektes Deutsch übersetzen.